content/docs/user-guide/interactivity/input/using-player-input.md

```diff
@@ -0,0 +1,145 @@
+
+![Adding Input Event Groups](/images/user-guide/interactivity/input/new-inputbindings-asset.png)
+
+1. To create a new input event group, click the {{< icon "add.svg" >}} icon.
+
+1. For **Event Name**, enter a name for the event, such as `Action`.
+
+
+### Creating event generators
+
+After you create an input event group, you can add event generators to the group. An *event generator* is a handler that generates the named event. For example, a pressed key, a held mouse button, or a series of actions on a game controller results in the named event.
+
+![Adding Event Generators](/images/user-guide/interactivity/input/new-inputbindings-asset-2.png)
+
+1. In the new input event group, click the {{< icon "add.svg" >}} icon next to **Event Generators**.
+
+1. In the **Class to create** dialog box, select `InputEventMap`, and then choose **OK**.
+
+1. Specify changes for the event generator. Each event generator has a set of properties that you can customize.
+
+### Setting event generator properties
+
+The following `.inputbindings` file specifies a keyboard for the **Input Device Type** and the space bar for the **Input Name**.
+
+   ![Input bindings configuration example in Asset Editor.](/images/user-guide/interactivity/input/inputbindings-example.png)
+
+### Saving your input bindings asset
+
+1. In the **Asset Editor**, choose **File, Save**.
+
+1. Enter a name for the `.inputbindings` file, and then choose **Save**.
+
+1. In Entity Inspector, in the Input component, for **Input to event bindings**, click {{< icon "browse-edit-select-files.svg" >}} and select your `.inputbindings` file.
+
+## Mapping input events to gameplay logic
+
+After you create an `.inputbindings` file and specify input events, you can use Script Canvas or Lua to map the input events to gameplay logic. You can create gameplay logic in a visual scripting environment with **Script Canvas Editor** or write your own Lua scripts with **Lua Editor** (Lua IDE).
+
+### Using input in Script Canvas 
+
+You can create a Script Canvas graph that connects to input events. For more information about scripting with Script Canvas, refer to [Creating Gameplay and Other Behaviors with Script Canvas](/docs/user-guide/scripting/script-canvas).
+
+1. In O3DE Editor, select the entity with the Input component in the viewport or **Entity Outliner**.
+
+1. In Entity Inspector, click **Add Component** and add the **Script Canvas** component.
+
+1. In the Script Canvas component, for **Script Canvas Asset**, click {{< icon "open-in-internal-app.svg" >}}, and then create a new Script Canvas graph like the following example.
+
+### Example Script Canvas graph
+
+In the following graph, the **InputHandler** node connects `Action` events to various **Transform** nodes. The Input component sends **Pressed**, **Held**, and **Released** events to the **InputHandler** node When the state of an event generator changes.
```

Comment:
```suggestion
In the following graph, the **InputHandler** node connects `Action` events to various **Transform** nodes. When the state of an event generator changes, the Input component sends **Pressed**, **Held**, and **Released** events to the **InputHandler** node.
```
Typo: Capital "When" in the middle of the sentence.

I still recommend moving the conditional phrase ("When the state...") to the front of the second sentence, where capitalizing it is fine.

If you prefer to keep that phrase at the end, be sure to make "when" lower case.